spec/helpers/set.js

```diff
@@ -0,0 +1,79 @@
+'use babel'
+
+import {setEqual} from '../../src/set-helpers'
+
+Set.prototype.isEqual = (other) => { // eslint-disable-line no-extend-native
```

Comment:
Avoid what? The eslint suppression, or the isEqual? Yes, this was converted verbatim, no additional things were added.